modules/terrain/docs/api-reference/quantized-mesh-loader.md

```diff
@@ -39,7 +39,7 @@ const data = await load(url, QuantizedMeshLoader, options);
 | Option                  | Type            | Default        | Description                                                                     |
 | ----------------------- | --------------- | -------------- | ------------------------------------------------------------------------------- |
 | `quantized-mesh.bounds` | `array<number>` | `[0, 0, 1, 1]` | Bounds of the image to fit x,y coordinates into. In `[minX, minY, maxX, maxY]`. |
-| `terrain.skirtHeight`   | `number`        | `null`         | If set, create the skirt for the tile with particular height                    |
+| `terrain.skirtHeight`   | `number`        | `null`         | If set, create the skirt for the tile with particular height in meters          |
```

Comment:
What if not set? Should the skirt height default to starting from ground (z = 0)?

```suggestion
| `terrain.skirtHeight`   | `number | null`        | `null`         | If set, create the skirt for the tile with particular height in meters          |
```

Reply:
If not set there won't be a skirt. Drawing a skirt to z=0 will give artifacts for areas below sea level.

Reply:
> If not set there won't be a skirt (same as existing behavior).

In that case, suggest the type should be `number` | `false`.

> Drawing a skirt to z=0 will give artifacts for areas below sea level.

Can we clarify what `height` means. This is the ground elevation level from which the skirt starts?

Reply:
> suggest the type should be `number` | `false`.

👍 

> Can we clarify what `height` means

`skirtHeight` is the distance that should be subtracted from the edge's elevation when creating the skirt. So if the edge vertex has an elevation value of `500` and `skirtHeight` is `50`, then new edge triangles will be created using a vertex directly below the existing edge vertex with an elevation value of `450`. (At least that's my understanding; @belom88 can confirm).

https://github.com/visgl/loaders.gl/blob/41c5a1d4f80e648f214026bbe4656c77bb311eb7/modules/terrain/src/lib/helpers/skirt.js#L133

Reply:
Why would we use relative heights as opposed to extruding all the way down to a flat base level?

Reply:
I suppose we could extrude down to a flat base level ([defaulting to `-430`](https://en.wikipedia.org/wiki/List_of_places_on_land_with_elevations_below_sea_level#Asia)), but I don't really see the need to instead of using relative heights. If you use a relative skirt height of `2 * meshMaxError`, it won't be possible to ever see a seam. In the `TerrainLoader` we could default the skirt height to `2 * meshMaxError`. In the `QuantizedMeshLoader`, the mesh max error is defined on the server side.

I'm also curious if one approach is better than the other for non-terrestrial applications, but don't know.